docs/content/installation/cloud.md

```diff
@@ -48,7 +48,7 @@ To login to a server provisioned by Flynn, you can specify the generated key and
 IP address of the server:
 
 ```text
-ssh -i ~/.flynn/installer/keys/flynn ubuntu@SERVER_IP
+ssh -i ~/.flynn/installer/keys/flynn root@SERVER_IP
```

Comment:
The user varies based on the VM provider, we should leave the default as `ubuntu` but advise users that it differs by provider:

* DigitalOcean: `root`
* Azure: `flynn`